pyLoadCore.py

```diff
@@ -639,8 +639,22 @@ def restart(self):
             except :
                 pass
 
-        execl(executable, executable, *sys.argv)
-        _exit(0)
+        if os.name == "nt":
+            if getattr(sys, "frozen", False):   # py2exe
+                args = [join(pypath, "restart_on_windows.exe")]
+            else:
+                args = [executable, join(pypath, "module", "lib", "restart_on_windows.py")]
+            args.append(str(os.getpid()))
+            args.extend(sys.argv)
+            try:
+                subprocess.Popen(args, close_fds=True)
```

Comment:
Codacy found an issue: [subprocess call - check for execution of untrusted input.](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=567903)